decksite/view.py

```diff
@@ -303,13 +303,10 @@ def prepare_leaderboards(self) -> None:
             self.prepare_leaderboard(l)
 
     def prepare_leaderboard(self, leaderboard: List[Container]) -> None:
```

Comment:
Alternative thought. Insist that callers attach a "score" tuple to each entry in `leaderboard` and do the logic once-only in here. Not sure if it's any better.

Reply:
Yeah... that makes sense. Should have realized to do that when I coded the kind of loop which checks for ties 3 times :P. I'll make that change.